chirp/drivers/uv5r.py

```diff
@@ -1635,34 +1670,20 @@ def apply_offset(setting, obj):
                                       STEP_LIST, STEP_LIST[_vfob.step]))
                 workmode.append(rs)
 
-        if _has_fmradio:
-            fm_preset = RadioSettingGroup("fm_preset", "FM Radio Preset")
-            group.append(fm_preset)
-
-            # broadcast FM settings
-            value = self._memobj.fm_presets
-            value_shifted = ((value & 0x00FF) << 8) | ((value & 0xFF00) >> 8)
-            if value_shifted >= 65.0 * 10 and value_shifted <= 108.0 * 10:
-                # storage method 3 (discovered 2022)
-                self._bw_shift = True
-                preset = value_shifted / 10.0
-            elif value >= 65.0 * 10 and value <= 108.0 * 10:
-                # storage method 2
-                preset = value / 10.0
-            elif value <= 108.0 * 10 - 650:
-                # original storage method (2012)
-                preset = value / 10.0 + 65
-            else:
-                # unknown (undiscovered method or no FM chip?)
-                preset = False
-            if preset:
-                rs = RadioSettingValueFloat(65, 108.0, preset, 0.1, 1)
-                rset = RadioSetting("fm_presets", "FM Preset(MHz)", rs)
-                fm_preset.append(rset)
-
         dtmf = RadioSettingGroup("dtmf", "DTMF Settings")
         group.append(dtmf)
 
+        # broadcast FM settings
+
+        # radios with the dropped byte issue also does not expose the broadcast
+        # FM frequency to CHIRP and ignores any frequency provided by CHIRP.
+        # Older radios that do expose the broadcast FM frequency to CHIRP have
+        # a minimum of 3 different know definitions for storing the
+        # frequencies. Sine some frequencies have colisions for some of the
```

Comment:
"since" and "collisions" are misspelled.

Reply:
Thanks